### PR TITLE
Create proc_creation_macos_usage_of_jamf.yml

### PR DESCRIPTION
**Pull request by gr00T0x**
### Summary of the Pull Request

To create a new process creation rule for MacOS for monitoring of malicious usage of the MDM Jamf CLI binary

### Detailed Description of the Pull Request / Additional Comments

This is to add a new process creation rule for the use of the MDM solution JAMF on MacOS. You can use this legitimate binary for multiple malicious purposes such as removing application restriction controls, creating user accounts and elevating privileges. There is also a Mythic C2 Agent written that utilises JAMF to blend in as legitimate traffic (Link can be found in Sigma rule)

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

**Comment by gr00T0x** [comment]
Detailed Description:

This is to add a new process creation rule for the use of the MDM solution JAMF on MacOS. You can use this legitimate binary for multiple malicious purposes such as removing application restriction controls, creating user accounts and elevating privleges. There is also a Mythic C2 Agent written that utilises JAMF to blend in as legitimate traffic (Link can be found in Sigma rule)

**Comment by gr00T0x** [comment]
Summary: To create a new process creation rule for MacOS for monitoring of malicious usage of the MDM Jamf CLI binary

**Comment by gr00T0x** [comment]
Apologies line 30 needs to be changed to: condition: selection_jamf and 1 of selection_param

**Comment by gr00T0x** [comment]
Anything needed my end for this rule?

**Review by nasbench — APPROVED** [review]

**Review by phantinuss — APPROVED** [review]

File: rules/macos/process_creation/proc_creation_macos_jamf_usage.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+title: JAMF MDM Execution
+id: be2e3a5c-9cc7-4d02-842a-68e9cb26ec49
+status: experimental
+description: |
+    Detects execution of the "jamf" binary to create user accounts and run commands. The binary can be abused by attackers on the system in order to bypass security controls or remove application control polices for example.
```
Comment (phantinuss) [inline rules/macos/process_creation/proc_creation_macos_jamf_usage.yml:5]:
```suggestion
    Detects execution of the "jamf" binary to create user accounts and run commands. For example, the binary can be abused by attackers on the system in order to bypass security controls or remove application control polices.
```

File: rules/macos/process_creation/proc_creation_macos_jamf_susp_child.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+detection:
+    selection:
+        ParentImage|endswith: '/jamf'
+        Image|endswith:
+            # Note: Add additional binaries/commands that are uncommon during your typical admin usage of Jamf
+            - '/bash'
+            - '/sh'
+    condition: selection
+falsepositives:
+    - Legitimate execution of custom scripts or commands by Jamf administrators. Apply additional filter accordingly
```
Comment (phantinuss) [inline rules/macos/process_creation/proc_creation_macos_jamf_susp_child.yml:25]:
```suggestion
    - Legitimate execution of custom scripts or commands by Jamf administrators. Apply additional filters accordingly
```

**Review by phantinuss — APPROVED** [review]

**Review by phantinuss — APPROVED** [review]

File: rules/macos/process_creation/proc_creation_macos_jamf_susp_child.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+title: JAMF MDM Potential Suspicious Child Process
+id: be2e3a5c-9cc7-4d02-842a-68e9cb26ec49
```
Comment (nasbench) [inline rules/macos/process_creation/proc_creation_macos_jamf_susp_child.yml:2]:
```suggestion
id: 2316929c-01aa-438c-970f-099145ab1ee6
```

**Pull request merged**